nemo/core/optim/optimizers.py

```diff
@@ -1,49 +1,33 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Revert. You cannot remove the license header

Reply:
Can you say more about the exact bugs this implementation will cause? Or are these just projections? Readibility is certainly non-trivial and a PR doesn't need a specific logical change to be non-trivial. This code was also optimized and this was noted at the PR.